code/game/machinery/portable_turret.dm

```diff
@@ -986,6 +986,11 @@
 		return 0
 	return 10
 
+/obj/machinery/porta_turret/syndicate/pod
+	health = 40
+	projectile = /obj/item/projectile/bullet/weakbullet3
+	eprojectile = /obj/item/projectile/bullet/weakbullet3

```

Comment:
i'm a bit worried about the balance of these bullets but meh, if they're too strong we can nerf them